libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -179,6 +182,9 @@ - (id) initWithConfig:(SFHybridViewConfig *) viewConfig useUIWebView:(BOOL) useU
     self = [super init];
     if (self) {
         self.useUIWebView = useUIWebView;
+        if (useUIWebView) {
```

Comment:
Not setting anything for `WKWebView`, since it's the default. The absence of `UI` indicates `WKWebView` is being used.